UPGRADE.md

```diff
@@ -2,6 +2,37 @@
 
 ## 2.2.17
 
+### Add missing `kernel.reset` event to document manager cache
```

Comment:
Add missing `kernel.reset` tag for document manager cache services

Reply:
```suggestion
### Add missing kernel.reset tag for document manager cache services
```